controller/space.go

```diff
@@ -415,6 +415,24 @@ func ConvertSpaceFromModel(ctx context.Context, db application.DB, request *goa.
 	relatedCollaboratorList := rest.AbsoluteURL(request, fmt.Sprintf("/api/spaces/%s/collaborators", spaceIDStr))
 	relatedFilterList := rest.AbsoluteURL(request, "/api/filters")
 
+	// fetch all categories links
+	categoriesData := []*app.GenericData{}
+	application.Transactional(db, func(appl application.Application) error {
```

Comment:
if there is an error inside this function, we will ignore it. Please capture the return value